libraries/lib-wave-track/WaveClip.cpp

```diff
@@ -394,6 +477,21 @@ bool WaveClip::HandleXMLTag(const std::string_view& tag, const AttributesList &a
                return false;
             SetTrimRight(dblValue);
          }
+         else if (attr == "rawAudioTempo")
+         {
+            if (!value.TryGet(dblValue))
+               return false;
+            if (dblValue == 0)
+               mRawAudioTempo.reset();
```

Comment:
If 0 is an excluded value, maybe the member doesn't need to be `std::optional` but instead you would test the value wherever it is used.

Reply:
One advantage of `optional`: the compiler will tell you if you are not de-referencing when you should. A bare double won't warn against unchecked uses.